src/app/home/avatar-generator/avatar-generator.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import htmlToImage from 'html-to-image';
+import {avatarOptions, buildAvatar} from './AvatarUtils';
+
+@Component({
+  selector: 'wgj-avatar-generator',
+  templateUrl: './avatar-generator.component.html',
+  styleUrls: ['./avatar-generator.component.scss'],
+})
+export class AvatarGeneratorComponent implements OnInit {
+  avatar = avatarOptions;
+  avatarKeys = Object.keys(avatarOptions);
+  constructor() { }
+  @ViewChild('screen') screen: ElementRef;
+  @ViewChild('canvas') canvas: ElementRef;
+  @ViewChild('downloadLink') downloadLink: ElementRef;
+
+  dowloadImage(): void {
```

Comment:
There is a small typo in this method name, I think you meant `downloadImage`

Reply:
fixed